docs/api.rst

```diff
@@ -23,6 +23,14 @@
 
 .. inheritance-diagram:: adafruit_displayio_layout.widgets.switch_round
 
+.. automodule:: adafruit_displayio_layout.widgets.dial
+   :members:
+   :member-order: bysource
+   :inherited-members:
+
+.. inheritance-diagram:: adafruit_displayio_layout.widgets.dial
+
 .. automodule:: adafruit_displayio_layout.widgets.icon_widget
    :members:
    :member-order: bysource
+
```

Comment:
I think it wants one of the newlines at the end of this file deleted. It's picky about having exactly 1 empty newline at the end. I made this change locally and it seems to pass actions after that.

Reply:
Thanks, for the note on this.  I updated and resubmitted.